web/components/templates/requestsV2/requestsPageV2.tsx

```diff
@@ -253,7 +256,9 @@ const RequestsPageV2 = (props: RequestsPageV2Props) => {
     isLive
   );
 
-  const requestWithoutStream = requests.find((r) => {
+  const { requestBodies: s3Bodies } = useGetS3Bodies(requestsSkeleton); // how the data ends up in the table??
```

Comment:
Can we rename this. These are not just s3Bodies. These are all the requests with s3bodies

Reply:
Can we also rename useGetS3Bodies

Reply:
Actually can we shove this into useRequestPageV2?

Reply:
Wait if I put it inside `useRequestPageV2` won't it behave like how it was before this pr?